examples/dt/uni07eta/control-plane.md

```diff
@@ -0,0 +1,88 @@
+# Configuring networking and deploy the OpenStack control plane
+
+## Assumptions
+
+- A storage class called `local-storage` should already exist.
+- Cluster observability operator is already deployed. If not, follow the
+  steps found [below](#cluster-observability-operator).
+
+### Cluster observability operator
+
+Cluster Observability Operator must be installed as it is required by OpenStack
+Telemetry operator. If not installed, the below steps can be followed
+
+```bash
+cat > subscription.yaml << EOF
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: observability-operator
+  namespace: openshift-operators
+  labels:
+    operators.coreos.com/observability-operator.openshift-operators: ""
+spec:
+  channel: development
+  installPlanApproval: Automatic
+  name: cluster-observability-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+EOF
```

Comment:
This is required for `telemetry` operator. Based on the `kustomization`, I noticed we are not enabling `telemetry` hence we could remove this lines.

Reply:
done